ui/src/message_popup/data_lists.jsx

```diff
@@ -1,171 +1,248 @@
 export const allStudents = [
   { 
     id: 1,
-    name: 'Kevin',
-    grade: '7th grade',
-    gender: 'male',
-    race: 'Hispanic',
-    behavior: 'Disruptive in class',
+    name: `Kevin`,
```

Comment:
This is good for consistency, but I'd propose that if there's no escaping necessary we use single quotes.  In general, we want most static strings to be static and only have ones be dynamic or escaped if they need to.

I'm not sure of a better reason *why* but I think simplicity is the best one.  Also performance here won't matter but it's possible it's slightly more work for the JS engine to interpret these than not.  In general though I think if it's a static string we should use quotes and if it needs escaping or interpretation then we use the ticks.